sdk/logs/src/main/java/io/opentelemetry/sdk/logs/GlobalLogEmitterProvider.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.logs;
+
+/** This class is a temporary solution until log SDK is marked stable. */
```

Comment:
Two thoughts:

1. Do we want a global accessor for the log sdk? There's not a precedent for providing global accessor for SDK components. `GlobalMeterProvider` provides access to `MeterProvider`, not `SdkMeterProvider`. `GlobalOpenTelemetry` provides access to `OpenTelemetry`, not `OpenTelemetrySdk`. Who would use this?

2.  Related to the javadoc: because there are no plans for a log api, this likely isn't a temporary solution.

Reply:
Ok, I see and It has been deleted; this can be defined in my business code in a private way